### ISSUE TITLE
MIG method

### ISSUE THREAD

**Issue by britney-code** [Trustworthy-AI-Group/TransferAttack#43]
Thank you for your great work, this framework has been really helpful to me. Do you have any plans to reproduce the code of this paper into your framework? "Transferable Adversarial Attack for Both Vision Transformers and Convolutional Networks via Momentum Integrated Gradients" in 2023 ICCV.
In fact, I have some issues with MIG's approach, so that I can't reproduce its results. For example, the loss function in IG is the logits of the corresponding category or cross entropy, etc.
